independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/parameters/converters/ArrayConverter.java

```diff
@@ -54,6 +54,7 @@ public static class ArraySupplier implements DelegatingParameterConverterSupplie
         public ArraySupplier() {
         }
 
+        // invoked by reflection for BeanParam in ClassInjectorTransformer
```

Comment:
Given this need, are these classes registered for reflection?

Reply:
This is not a new behaviour, I only added the docs after I noticed that the constructor was being called this way by existing code.